### PR TITLE
Add rel="alternate" to JSON representation

### PR DESCRIPTION
**Pull request by BigBlueHat**
This can be useful for developers wanting to discover the JSON representation from the HTML one. I intend to set another rel="alternate" going the other way...once I find where that lives...

### CODE REVIEW

**Comment by BigBlueHat** [comment]
Oh! This also includes some necessary fixes to `link_attrs` code and it's location. Happy to split these into two PRs if need be.

File: h/templates/layouts/base.html [hunk 1]
```diff
@@ -5,16 +5,16 @@
       <meta charset="UTF-8" />
       <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1"/>
       <meta name="viewport" content="width=device-width,initial-scale=1" />
-      {% for attrs in meta_attrs -%}
```
Comment (aron) [inline h/templates/layouts/base.html]: Why'd you get rid of the nice whitespace syntax? :(

File: h/views.py [hunk 2]
```diff
@@ -26,14 +26,19 @@ def annotation(context, request):
     title = 'Annotation by {user} on {title}'.format(
         user=context['user'].replace('acct:', ''),
         title=context['document']['title'])
```
Comment (aron) [inline h/views.py]: While you're here would you also fix this line, not all annotations are guaranteed to have the document property so we need a fallback if it's missing.

File: h/templates/layouts/base.html [hunk 3]
```diff
@@ -5,16 +5,16 @@
...
+      {% for attrs in meta_attrs %}
         <meta {% for key, value in attrs.items() %}{{ key }}="{{ value }}" {% endfor %}/>
-      {% endfor -%}
+      {% endfor %}
+      {% for attrs in link_attrs %}
+        <link {% for key, value in attrs.items() %}{{ key }}="{{ value }}" {% endfor %}/>
+      {% endfor %}
```
Comment (aron) [inline h/templates/layouts/base.html]: Also, not sure the links should have been moved into the `meta` block. I'd prefer to keep them with the other links.

File: h/templates/layouts/base.html (hunk 1 above)
Comment (BigBlueHat) [inline h/templates/layouts/base.html]: Is that what that cute little dash randomly attached to the side of the % does? Neat! :stuck_out_tongue_winking_eye: 

Sorry...new to Jinja. More familiar with Mustach and Handlebars.

Thanks @aron. Will fix.

File: h/templates/layouts/base.html (hunk 3 above)
Comment (BigBlueHat) [inline h/templates/layouts/base.html]: Yeah. Moved it back. Worked fine. Had thought it was broken...but that was apparently something else. My bad.

**Comment by BigBlueHat** [comment]
@aron think I fixed all the things. :smile_cat:

**Comment by aron** [comment]
@BigBlueHat did you actually push all the things too?

**Comment by BigBlueHat** [comment]
:frowning:

**Comment by BigBlueHat** [comment]
@aron this time I did :exclamation:  :triumph:

**Pull request merged**